.gitignore

```diff
@@ -60,3 +60,5 @@ yarn-debug.log*
 
 # Ignore vim swapfiles
 # .*.swp
+
+vendor/bundle
```

Comment:
@acherukuri , this is _might_ be a vestige of your PowerHRG bundle cache config - it might not be relevant to this project. Had you considered checking `~/.bundle/config` to see if `BUNDLE_PATH` is set globally to `vendor/bundle`? Most projects that aren't PowerHRG projects will let bundler install gems into the GEM_PATH instead of providing a custom `BUNDLE_PATH` as PowerHRG does.

Reply:
@indiebrain, I have installed gems to `vendor/bundle` on purpose as I'm using VS Code and it is very good with indexing ruby gems from `vendor/bundle` folder. It's handy for browsing code and documentation.

Reply:
@acherukuri You should have that in your personal global git ignore.

Reply:
@terryfinn, Thanks for reminding that. I have totally forgotten about it.